### PR TITLE
Add organization.user_ids and user.organization_id relation

### PR DESCRIPTION
**Pull request by reiterl**
Update the tests, and add test parts.
Add a migration.

### CODE REVIEW

File: openslides_backend/action/actions/meeting/import_.py [hunk 1]
```diff
@@ -549,9 +549,7 @@ def create_events(
                                 list_fields["add"][field] = value
                             else:
                                 fields[field] = value
-                        elif isinstance(
-                            model_field, (RelationListField, RelationField)
-                        ):
+                        elif isinstance(model_field, RelationListField):
```
Comment (r-peschke) [inline openslides_backend/action/actions/meeting/import_.py:552]: What's about RelationField?

File: openslides_backend/action/actions/meeting/import_.py [hunk 2]
```diff
@@ -631,6 +629,26 @@ def append_extra_events(
                 )
             )
 
+        # add new users to the organization.user_ids
+        new_user_ids = []
+        for user_entry in json_data.get("user", {}).values():
```
Comment (r-peschke) [inline openslides_backend/action/actions/meeting/import_.py:634]: Are you sure, that the user_ids from json_data are the new ones or still the old ones from export?

**Review by r-peschke — COMMENTED** [review]
Some points to check and we have to resolve with @ostcar the restriction mode F problem

File: openslides_backend/action/actions/meeting/import_.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:552]: Does not work and is not needed here and lead to errors.

File: openslides_backend/action/actions/meeting/import_.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:634]: I except the json_data["user"] to be merged ones and new ones. And then use only the new ones.

File: global/meta/models.yml [hunk 3]
```diff
@@ -433,6 +437,11 @@ user:
     description: Calculated. All ids from group_$_ids as integers.
     read_only: true
     restriction_mode: E
+  organization_id:
+    type: relation
+    to: organization/user_ids
+    required: True
+    restriction_mode: F
```
[the remark this answers is not in the document]
Comment (ostcar) [reply global/meta/models.yml:444]: In this case F is correct. This means that you need the OML can_manage_users.

File: global/meta/models.yml [hunk 4]
```diff
@@ -147,6 +147,10 @@ organization:
     to: mediafile/owner_id
     on_delete: CASCADE
     restriction_mode: A
+  user_ids:
+    type: relation-list
+    restriction_mode: A
```
Comment (ostcar) [inline global/meta/models.yml:152]: This is the place, where for performance reasons I would like you to use another restriction_mode.

File: openslides_backend/action/actions/meeting/import_.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:634]: Perhaps here is an error. I will look at it again.

File: openslides_backend/action/actions/meeting/import_.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:552]: The handling of the RelationField should still be in place.

File: openslides_backend/action/actions/meeting/import_.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/meeting/import_.py:634]: Ah, there was a bug, I changed the code.

File: tests/system/action/meeting/test_import.py [hunk 5]
```diff
@@ -1742,7 +1765,7 @@ def test_all_migrations(self) -> None:
         with CountDatastoreCalls(verbose=True) as counter:
             response = self.request("meeting.import", data)
         self.assert_status_code(response, 200)
-        assert counter.calls == 6
+        assert counter.calls == 7
```
Comment (r-peschke) [inline tests/system/action/meeting/test_import.py:1768]: This is the test to test all migrations. Please remove the organization_id in instance data (`del data["meeting"]["user"]["1"]["organization_id"]` and insert a test that it was set by migration (`self.assert_model_exists("organization/1", {"user_ids": [1, 2], "active_meeting_ids": [1, 2]})`)

**Review by r-peschke — CHANGES_REQUESTED** [review]
Still want to check whether the search works with the autoupdate restriction modes and https://github.com/OpenSlides/openslides-backend/issues/1459

**Review by ostcar — CHANGES_REQUESTED** [review]
See my comment from 5 days ago. The restriction_mode of the field organization/user_ids should be changed

File: global/meta/models.yml (hunk 4 above)
Comment (reiterl) [reply global/meta/models.yml:152]: I chatted with @r-peschke about this topic. I will change the restriction_mode of organization/user_ids to C.

File: tests/system/action/meeting/test_import.py (hunk 5 above)
Comment (reiterl) [reply tests/system/action/meeting/test_import.py:1769]: Done.

File: global/meta/models.yml (hunk 4 above)
Comment (reiterl) [reply global/meta/models.yml]: Done.

**Review by r-peschke — APPROVED** [review]
Please write a test for the change in user/create.py

**Pull request merged**